src/qml/README.md

```diff
@@ -6,9 +6,17 @@ This directory contains the source code for an experimental Bitcoin Core graphic
 
 Insecure CI artifacts are available for local testing of the master branch, avoiding the need to build:
 - for Windows: [`insecure_win_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip)
-- for macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
+- for Intel macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
+- for Apple Silicon macOS: [`insecure_mac_arm64_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_arm64_gui.zip)
 - for Android: [`insecure_android_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip)
 
+Note: For Apple Silicon macOS machines, the binary must be signed before it can
+be ran. To apply a signature, run the following on the untarred CI artifact:
```

Comment:
```suggestion
be ran. To apply a signature, run the following on the unzipped CI artifact:
```

It's more of a question than a suggestion. Since the CI artifacts are in `.zip` format, shouldn't we use **unzipped** over **untarred**?

Reply:
done